src/dali_model.cc

```diff
@@ -25,16 +25,17 @@
 namespace triton { namespace backend { namespace dali {
 
 TRITONSERVER_Error* DaliModel::Create(TRITONBACKEND_Model* triton_model, DaliModel** state) {
-  TRITONSERVER_Error* error = nullptr;  // success
   try {
     *state = new DaliModel(triton_model);
+    return nullptr;
+  } catch (TritonError& e) {
+    return e.release();
```

Comment:
who deletes this?

Reply:
Triton server. The error is returned and propagated up to the server which handles it.